tomviz/DataSource.cxx

```diff
@@ -66,13 +66,19 @@ class DataSource::DSInternals
     vtkFieldData* fd = data->GetFieldData();
     if (!this->TiltAngles)
     {
+      if (!fd->HasArray("tilt_angles"))
```

Comment:
I've deleted it, and updated the branch.  I think this was left over from the first implementation of this branch.  I copied the contents out into the containing block and I guess I forgot to delete this.